final_task/rss_reader/functions/convertation.py

```diff
@@ -6,16 +6,108 @@
 
 from ebooklib import epub
 import os
+import requests
 import dominate
 import dominate.tags as tag
 
 
+def set_internet_flag(logger):
+    """If Internet connection, set flag to True, else False"""
+    try:
+        internet = check_internet_connection(logger)
+    except InternetConnectionError:
+        internet = False
+    finally:
+        return internet
```

Comment:
You do not really need to put this under `finally` block here. Is case of exception, the interpreter will run code inside the `except` block and then will run the code after it.

Reply:
I know, I didn't want to write return internet two times

Reply:
And why would you need to write `return internet` two times?

```
try:
    internet = ...
except SomeException:
    internet = ...
return internet
```

Reply:
Yes, i was wrong